File: trójkąt.py
from math import sqrt
a = int(input("wprowadz a"))
b = int(input("wprowadz b"))
c = int(input("wprowadz c"))

if a > b and a > c:
    if b + c > a:
        p = (a + b + c) / 2
        Pole = sqrt(p * (p - a) * (p - b) * (p - c))
        if (c*c) + (b*b) == (a*a):
            print(Pole, "Trójkąt prostokątny")
        elif (c*c) + (b*b) > (a*a):
            print(Pole, "Trójkąt ostrokątny")
        else:
            print(Pole, "Trójkąt rozwartokątny")
elif b > a and b > c:
    if a + c > b:
        p = (a + b + c) / 2
        Pole = sqrt(p * (p - a) * (p - b) * (p - c))
        if (c*c) + (a*a) == (b*b):
            print(Pole, "Trójkąt prostokątny")
        elif (c*c) + (a*a) > (b*b):
            print(Pole, "Trójkąt prostokątny")
        else:
            print(Pole, "Trójkąt rozwartokątny")
elif c > a and c > b:
    if b + a > c:
        p = (a + b + c) / 2
        Pole = sqrt(p * (p - a) * (p - b) * (p - c))
        if (a*a) + (b*b) == (c*c):
            print(Pole, "Trójkąt prostokątny")
        elif (a*a) + (b*b) > (c*c):
            print(Pole, "Trójkąt prostokątny")
        else:
            print(Pole, "Trójkąt rozwartokątny")

File: trójkąt.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int a, b, c;
int p, Pole;

void pole_trojkata (int a, int b, int c, int Pole, int p){
	 p = (a + b + c) / 2;
	 printf ("Podaj a b c");
	scanf("%d %d %d", &a, &b, &c);
    Pole = sqrt(p * (p - a) * (p - b) * (p - c));
   
    return  printf ("Jest trojkatem a pole wynosi %d" ,Pole);
    }
int main(  ) {
	
	while (a > 0 & b > 0 & c > 0 ){
	if (a + b > c) {
		pole_trojkata(a,b,c);
		
		
	} 
	else{
		printf("Nie jest trojkatem");
		};
}
	return 0;
}
